src/database/schemas/deliverable.schema.ts

```diff
@@ -0,0 +1,40 @@
+import { Prop, Schema, SchemaFactory } from '@nestjs/mongoose';
+import { Document, Types } from 'mongoose';
+import { nanoid } from '../functions/nano-id.function';
+
+export type DeliverableType = Deliverable & Document;
+
+@Schema({ timestamps: true })
+export class Deliverable {
+    _id: Types.ObjectId;
+
+    @Prop({
+        type: String,
+        default: () => nanoid(),
+        unique: true,
+    })
+    id: string;
+
+    @Prop({
+        type: String,
+        required: true,
+        trim: true,
+        maxLength: 256,
+    })
+    title: string;
+
+    @Prop({
+        type: String,
+        required: true,
+        trim: true,
```

Comment:
issue(blocking): Add `maxLength` here as we do not risk that someone floods our database.

Reply:
is used 4089 since i am not sure how long this field can actually get